src/pages/storage/forms/StoragePoolFormMenu.tsx

```diff
@@ -28,6 +38,11 @@ const StoragePoolFormMenu: FC<Props> = ({ active, setActive }) => {
       <nav aria-label="Storage pool form navigation">
         <ul className="p-side-navigation__list">
           <MenuItem label={MAIN_CONFIGURATION} {...menuItemProps} />
+          <MenuItem
```

Comment:
Hmmm this would go against existing behaviour on other forms where sections are disabled and never hidden. In any case, I think having the section visible but disabled indicates to the users that there are additional settings for ceph storage pools, which isn't bad?

Reply:
True, I'm fine keeping it.
We do indeed hide sections, i.e. on network creation/editing when you disable ipv4 or ipv6, the menu sections get removed. We sometimes disable them, because the users needs to first fill out something on the main form and then the other sections get unlocked.
I am not sure what is best option here, but seems like we should make things more consistent.

Reply:
Aaah I see, that makes sense then. Let's hide the Ceph section and disable it if storage name isn't specified